### ISSUE TITLE
Enhance the API to support smart contract

### ISSUE THREAD

**Issue by semuxgo** [semuxproject/semux-core#56]
Depends on #53 

We need to expose some API for developers to create/call smart contracts on blockchain.

**Comment by orogvany** [comment]
the VM PR has call/create API calls

but i think this can be replaced with #72

We not just want API for calls, but also want to enable them to use existing tools

**Comment by orogvany** [comment]
Closing in favor of #72
